### PR TITLE
Returns the status codes in the report responses

### PR DESCRIPTION
**Pull request by andrysds**
for issue #9

### CODE REVIEW

**Comment by nitishm** [comment]
@andrysds Can you fix the failures reported in Travis. 
```
$ make validate
golangci-lint run
internal/vegeta/report.go:131:16: Error return value of `json.Unmarshal` is not checked (errcheck)
	json.Unmarshal(buf.Bytes(), &m)
	              ^
internal/vegeta/report.go:135:16: Error return value of `json.Unmarshal` is not checked (errcheck)
	json.Unmarshal(osd, &oldStatusCodes)
	              ^
internal/vegeta/report.go:148:29: Error return value of `(*encoding/json.Encoder).Encode` is not checked (errcheck)
	json.NewEncoder(buf).Encode(m)
	                           ^
make: *** [validate] Error 1
The command "make validate" exited with 2.
```

File: README.md [hunk 1]
```diff
@@ -229,6 +241,13 @@ curl http://localhost:8000/api/v1/report/
             "rate": 50.08399085265991,
             "requests": 500,
             "success": 1,
+            "success": 1,
```
Comment (nitishm) [inline README.md]: `success` is duplicated. Can you remove one of them.

File: internal/vegeta/report.go [hunk 2]
```diff
@@ -120,5 +121,36 @@ decode:
 		return "", err
 	}
```
Comment (nitishm) [inline internal/vegeta/report.go]: [Add comment]
// Adapt vegeta status code (type: map) to list of type: statusCode{}, as defined in swagger spec

File: internal/vegeta/report.go [hunk 3]
```diff
@@ -120,5 +121,36 @@ decode:
...
+
 	return buf.String(), nil
 }
+
+func modifyReportStatusCodes(buf *bytes.Buffer) error {
+	var m map[string]interface{}
+	if err := json.Unmarshal(buf.Bytes(), &m); err != nil {
+		return err
+	}
+
```
Comment (nitishm) [inline internal/vegeta/report.go]: [add comment]
// Read old status codes from vegeta report

File: internal/vegeta/report.go [hunk 4]
```diff
@@ -120,5 +121,36 @@ decode:
...
+	var oldStatusCodes map[string]int
+	osd, _ := json.Marshal(m["status_codes"])
+	if err := json.Unmarshal(osd, &oldStatusCodes); err != nil {
+		return err
+	}
+
```
Comment (nitishm) [inline internal/vegeta/report.go]: [add comment]
// Adapt status codes to type: statusCode{} and append to list.

**Comment by nitishm** [comment]
Great job @andrysds !! Thanks ! 🎉

**Comment by andrysds** [comment]
Thanks @nitishm !! 🎉

**Pull request merged**